src/bitmap/backend/atomic_bitmap.rs

```diff
@@ -225,7 +226,7 @@ mod tests {
 
     #[test]
     fn test_bitmap_impl() {
-        let b = AtomicBitmap::new(0x2000, 128);
+        let b = AtomicBitmap::new(0x2000, DEFAULT_PAGE_SIZE);
```

Comment:
nit: The helper variable probably could have been introduced in a separate commit.

Reply:
I think this doesn't warrant an additional commit.